app/components/pages/SubmissionWizard/steps/Editors/EditorsContent.js

```diff
@@ -0,0 +1,291 @@
+import React from 'react'
+import { Box } from 'grid-styled'
+import { Action, ErrorText } from '@pubsweet/ui'
+import MoreButton from '../../../../ui/molecules/MoreButton'
```

Comment:
This looks pretty ugly - can we not add the UI components to the import path somehow?

Reply:
We can create our own UI package in `client`. So this would then be `import MoreButton from '@elifesciences/pubsweet-ui'` or similar.

Reply:
Sounds a great suggestion - Let me open an issue for it :)